data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/response/CertificateRetrievalRpcResponse.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "record",
+  "name": "CertificateRetrievalRpcResponse",
+  "namespace": "net.corda.data.certificates.rpc.response",
+  "doc": "RPC operations response with a certificate chain.",
+  "fields": [
+    {
+      "name": "certificate",
+      "doc": "The certificates in PEM format.",
+      "type": [
+        "string",
```

Comment:
Same question as above. Is this intended to be a concatenated string of certs? I think we're using arrays in others places.